main.cpp

```diff
@@ -0,0 +1,103 @@
+
+#include <iostream>
+#include <vector>
+#include <cstring>
+#include <map>
+#include <algorithm>
+#include "zpool.h"
+
+using namespace std;
+
+
+int main() 
+{
+	bool ret;
+	int option;
+	char consent;
+	Zpool obj;
+
+	cout << "1: Create disk pool" << endl;
+	cout << "2: List all disk pools" << endl;
+	cout << "3: Add disk to the specified disk pool" << endl;
+	cout << "4: Get list of disks in the specified disk pool" << endl;
+	cout << "5: Remove disk from the specified disk pool" << endl;
+	cout << "6: Destroy pool" << endl ;
+	cout << "Enter the desired option from 1 to 6" << endl << endl;
+
+	cin >> option;
+
+
```

Comment:
Be consistent about using blank lines in the code. Stick to one blank line for separation and readability.